### DISCUSSION TITLE
How to change the message flag to whisper user?

### DISCUSSION THREAD

**Discussion by echan00** [Rapptz/discord.py#6171] [General]
I heard message flag value 1 << 6 will allow whispering to the user, how do I change the message flag to try this?

**Comment by EvokerKing** [comment]
I am guessing you mean DM them. You don't actually have to change the message flag to do this. To send a DM just simply do:
```py
userID = ctx.author.mention.replace('<@', '').replace('>', '')
user = ctx.get_user(int(userID))
await user.send(message)
```
This will get the user that ran a command, and then send them the `message` variable.

**Reply by echan00** [reply]
Thanks, I was actually referring to whisper within the same channel, but I now found out that you can only do this to respond to slash commands

**Comment by Rapptz** [comment]
This is only available to slash commands/interactions and not regular bots.
